boutiques_descriptors/deepprep_24_1_2.json

```diff
@@ -23,28 +23,133 @@
             "type": "String",
             "value-key": "<output_dir>"
         },
+        {
+            "description": "FreeSurfer license file",
+            "id": "fs_license_file",
+            "name": "fs_license_file",
+            "optional": false,
+            "type": "File",
+            "value-key": "<fs_license_file>"
+        },
         {
             "command-line-flag": "--bold_task_type",
             "description": "The task label of BOLD images",
             "id": "bold_task_type",
             "name": "bold_task_type",
             "optional": false,
             "type": "String",
-            "value-key": "<bold_task_type>",
+            "list": false,
+            "value-key": "<--bold_task_type>"
```

Comment:
are you sure it is mandatory param?

Reply:
yes it is according to the doc: 

https://deepprep.readthedocs.io/en/latest/usage_local.html#command-line-arguments